app/src/main/kotlin/com/simplemobiletools/gallery/pro/dialogs/ChangeFileThumbnailStyleDialog.kt

```diff
@@ -59,13 +65,28 @@ class ChangeFileThumbnailStyleDialog(val activity: BaseSimpleActivity) : DialogI
 
     override fun onClick(dialog: DialogInterface, which: Int) {
         config.fileRoundedCorners = binding.dialogFileStyleRoundedCorners.isChecked
-        config.animateGifs = binding.dialogFileStyleAnimateGifs.isChecked
+        if (binding.dialogFileStyleRoundedCorners.isChecked) {
+            config.animateGifs = false
+        } else {
+            config.animateGifs = binding.dialogFileStyleAnimateGifs.isChecked
+        }
         config.showThumbnailVideoDuration = binding.dialogFileStyleShowThumbnailVideoDuration.isChecked
         config.showThumbnailFileTypes = binding.dialogFileStyleShowThumbnailFileTypes.isChecked
         config.markFavoriteItems = binding.dialogFileStyleMarkFavoriteItems.isChecked
         config.thumbnailSpacing = thumbnailSpacing
     }
 
+    private fun updateAnimateGifsCheckbox() {
+        if (binding.dialogFileStyleRoundedCorners.isChecked) {
+            binding.dialogFileStyleAnimateGifs.alpha = 0.5f
+            binding.dialogFileStyleAnimateGifs.isChecked = false
+            binding.dialogFileStyleAnimateGifsHolder.setOnClickListener { Toast.makeText(activity, R.string.unsupported_animate_gifs_and_rounded_corners, Toast.LENGTH_SHORT).show() }
```

Comment:
Let me know what you think about this behaiour. If "rounded corners" is selected the option for "animate thumbnails" is grayed out and if pressed a toast is shown stating that rounded corners and animated thumbnails are not supported at the same time.
This is due to glide not supporting fit center and rounded corners at the same time, this would break the design and looks bad.